### DISCUSSION TITLE
Created a Subtitles Plugin For remotion

### DISCUSSION THREAD

**Discussion by ahgsql** [remotion-dev/remotion#3675] [Show and tell]
hey folks, i've just published my remotion-subtitle plugin to create subtitle captions with ease.

All you need is a string in srt format. 

https://github.com/ahgsql/remotion-subtitles

I m using stable-ts library to create srt files easly, then just copy contents of srt file and use it with my plugin.

```python
import stable_whisper
model = stable_whisper.load_model('base')
result = model.transcribe('audio.mp3')
result.to_srt_vtt('audio.srt',segment_level=False)```

Feel free to contribute my project for further Improvements

**Comment by mnlbox** [comment]
Hi @ahgsql ,

Really nice project and I like the animations that you already added there. I'm also working to add some text animations to my project. Maybe related to this discussion:
https://github.com/orgs/remotion-dev/discussions/3647

BTW, I highly recommend to check the link below to get some inspiration for even more different text animations, it's a list contains a huge number of text animations:
https://www.sliderrevolution.com/resources/css-text-animation/
